cmd/apps/inletsoperator_app.go

```diff
@@ -114,22 +115,30 @@ func MakeInstallInletsOperator() *cobra.Command {
 		}
 
 		tokenFileName, _ := command.Flags().GetString("token-file")
-
-		if len(tokenFileName) == 0 {
-			return fmt.Errorf(`--token-file is a required field for your cloud API token or service account JSON file`)
+		accessKey, _ := command.Flags().GetString("access-key")
+
+		var access_key_from, access_key_value string
+		if len(tokenFileName) > 0 {
+			access_key_from = "--from-file"
+			access_key_value = tokenFileName
+		} else if len(accessKey) > 0 {
+			access_key_from = "--from-literal"
+			access_key_value = accessKey
+		} else {
+			return fmt.Errorf(`--token-file or --access-key is a required field for your cloud API token or service account JSON`)
 		}
 
 		res, err := k8s.KubectlTask("create", "secret", "generic",
 			"inlets-access-key",
 			"--namespace="+namespace,
-			"--from-file", "inlets-access-key="+tokenFileName)
+			access_key_from, "inlets-access-key="+access_key_value)
```

Comment:
Please use camelCase for `access_key_from` -> accessKeyFrom etc. Same for `access_key_value`

Reply:
Btw, we use `token` and `token-file` to mean `access-key`, so do we need this change above?

Reply:
Updated the naming style.

Regarding the `--token-file`, it accepts only from file. There is no `--token`.

I added this `--access-key` so that we can pass the value as literal string instead of file. The name `--access-key` being used because the k8s secret key being used in inlets-operator is `inlets-access-key`.